<?php

namespace Factorial
use Factorial\Main;

try {
    Main::run();
} catch (Exception $exception) {

}